extensions/frontend-tiles/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@itwin/frontend-tiles",
+  "version": "4.0.0-dev.44",
+  "description": "Experimental alternative technique for visualizing the contents of iModels",
+  "main": "lib/cjs/frontend-tiles.js",
+  "module": "lib/esm/frontend-tiles.js",
+  "typings": "lib/cjs/frontend-tiles",
+  "license": "MIT",
+  "scripts": {
+    "build": "npm run -s copy:public && npm run -s build:cjs",
+    "build:ci": "npm run -s build && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "docs": "",
+    "extract-api": "betools extract-api --entry=frontend-tiles",
+    "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
+    "test": "",
+    "cover": ""
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/itwinjs-core/tree/master/core/frontend-tiles"
+  },
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "UI",
+    "Widget"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "dependencies": {
```

Comment:
```suggestion
  "peerDependencies": {
```

Reply:
you'll also want to add the rest of these deps to devDeps

Reply:
done, please review.